lib/irb/cmd/measure.rb

```diff
@@ -12,32 +12,29 @@ def initialize(*args)
         super(*args)
       end
 
-      def execute(type = nil, arg = nil, &block)
+      def execute(type = nil, arg = nil)
         # Please check IRB.init_config in lib/irb/init.rb that sets
         # IRB.conf[:MEASURE_PROC] to register default "measure" methods,
         # "measure :time" (abbreviated as "measure") and "measure :stackprof".
+
+        if block_given?
+          warn 'Configure IRB.conf[:MEASURE_PROC] to add custom measure methods.'
+          return
+        end
+
         case type
         when :off
-          IRB.conf[:MEASURE] = nil
           IRB.unset_measure_callback(arg)
         when :list
           IRB.conf[:MEASURE_CALLBACKS].each do |type_name, _, arg_val|
             puts "- #{type_name}" + (arg_val ? "(#{arg_val.inspect})" : '')
           end
         when :on
-          IRB.conf[:MEASURE] = true
-          added = IRB.set_measure_callback(type, arg)
+          added = IRB.set_measure_callback(nil, nil)
```

Comment:
`set_measure_callback(:on, nil)` does nothing (unless `IRB.conf[:MEASURE_PROC][:ON]` is set)
IRB was using "TIME" when `IRB.conf[:MEASURE] = true` is set and measure callback is not set.

Fixes this bug:
```
irb> measure :on
# Message `"TIME" is added.` is not show
irb> 1
processing time: 0.000032s
=> 1 # but "TIME" is added
```